src/components/CommentItem.vue

```diff
@@ -79,6 +80,21 @@ export default {
             nextpage: null,
         };
     },
+    mounted() {
+        const thisComment = this;
+        this.$el.querySelectorAll("a").forEach(elem => {
+            if (elem.innerText.match(/(?:[\d]{1,2}:)?(?:[\d]{1,2}):(?:[\d]{1,2})/)) {
+                elem.addEventListener("click", function (event) {
+                    if (!event || !event.target) return;
+                    if (!event.target.getAttribute("href").match(/(?<=t=)\d{1,}/)) return;
+
+                    const time = parseInt(event.target.getAttribute("href").match(/(?<=t=)\d{1,}/)[0]);
+                    thisComment.$emit("seek", time);
+                    event.preventDefault();
+                });
+            }
+        });
```

Comment:
Why can't we add just one event listener in the document/window and check if the element matches? (Through the concept of Event Bubbling)